packages/orchestrator/src/execution.ts

```diff
@@ -0,0 +1,63 @@
+import * as grpc from 'grpc'
+import * as base58 from '@mesg/api/lib/util/base58'
+import * as encoder from '@mesg/api/lib/util/encoder'
+import { Client } from './client'
+import * as Type from './typedef/execution'
+import * as API from './api/execution'
+
+// TODO: To improve using the types but if include the type, the definition is still included in the js
+export const Status: { [key: string]: Type.mesg.types.Status } = {
+  Unknown: 0, // Type.mesg.types.Status.Unknown,
```

Comment:
Read the todo, there is some weird behavior with enums in typescript, the file stays included when compiled even if it doesn't generate any js... maybe a bug in typescript